### PR TITLE
DCD-115: Update Aurora language to use "Amazon Aurora" instead of "AWS Aurora"

### PR DESCRIPTION
**Pull request by bpar476**

### CODE REVIEW

**Review by theghostwhoforks — APPROVED** [review]

**Review by ddomingorh — APPROVED** [review]
LGTM

**Comment by bpar476** [comment]
Corresponding PR here: https://github.com/atlassian/quickstart-atlassian-services/pull/5

Going to wait until my test deploys succeed and then merge 👍

**Pull request merged**
